### PR TITLE
Removes the magic in Input, reducing it to a styled TextField

### PR DESCRIPTION
**Pull request by rileyjbauer**
`Input` previously took its parent's instance as a prop in order to automatically update the field in its parent's state using just the field name, however, this caused an additional copy of the instance to be made for each `Input` element in the parent.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/150)

### CODE REVIEW

File: frontend/src/pages/NewRun.tsx [hunk 1]
```diff
@@ -140,14 +144,16 @@ class NewRun extends Page<{}, NewRunState> {
...
+            onChange={this.handleChange('description')} value={description} height='auto' />
 
           {experimentName && (
             <div>
               <div>This run will be associated with the following experiment</div>
-              <Input label='Experiment' instance={this} disabled={true} field='experimentName' />
+              <Input label='Experiment' onChange={this.handleChange('experimentName')}
+                disabled={true} value={experimentName} />
             </div>
           )}
```
[the remark this answers is not in the document]
Comment (rileyjbauer) [reply frontend/src/pages/NewRun.tsx]: I'll make this change in the branch where I'm working on `NewRun` tests

**Comment by yebrahim** [comment]
I saw all tests finish successfully on this. For a few minutes, I could see Github's three green buttons here and could merge manually, then saw this test failure. @IronPan any ideas what's happening here?

**Pull request merged**